### ISSUE TITLE
Form control has axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly#2530]
- IDs of active elements must be unique (3 form fields)

**Comment by jessiehuff** [comment]
This issue appears to be solved. 🙂
